exampleCourse/questions/template/multiple-choice/none-of-the-above/info.json

```diff
@@ -0,0 +1,8 @@
+{
+  "uuid": "d786ae4b-8b8b-4e40-9054-926e3464bb79",
+  "title": "Multiple choice: None of the above",
+  "topic": "Template",
+  "type": "v3",
+  "shareSourcePublicly": true,
+  "singleVariant": true
```

Comment:
Checking my understanding: we set this because the question isn't randomized and we wouldn't want a "new variant" button since that would just give exactly the same variant, is that correct?

Reply:
Correct, the same answer choices are always displayed.